src/main/scala/org/deepdive/ddlog/DeepDiveLogParser.scala

```diff
@@ -158,11 +165,12 @@ class DeepDiveLogParser extends JavaTokenParsers {
 
   def operator = "||" | "+" | "-" | "*" | "/" | "&"
   def typeOperator = "::"
-  val aggregationFunctions = Set("MAX", "SUM", "MIN", "ARRAY_ACCUM", "ARRAY_AGG")
+  val aggregationFunctions = Set("MAX", "SUM", "MIN", "ARRAY_ACCUM", "ARRAY_AGG", "COUNT")
 
-  // expressions
+  // expression
   def expr : Parser[Expr] =
-    ( lexpr ~ operator ~ expr ^^ { case (lhs ~ op ~ rhs) => BinaryOpExpr(lhs, op, rhs) }
+    ( "_" ^^ { case _ => Placeholder() }
```

Comment:
Yeah that sounds good.  In that other PR let's revise the grammar clearly to allow only exprs in the head and patts in body.